databases/week2/Part3.sql

```diff
@@ -0,0 +1,46 @@
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.title AS Task,user.email
+FROM task 
+JOIN user_task 
+ON task.id = user_task.task_id
```

Comment:
I would keep the ON on the same line as the JOIN, imho it reads better.

Reply:
I was totally confused while formatting queries. Thanks for the comments.